hw/mcu/nordic/pkg.yml

```diff
@@ -26,14 +26,101 @@ pkg.keywords:
 
 pkg.type: sdk
 
+pkg.ign_files.BSP_NRF52:
+    - "nrfx_adc.c"
```

Comment:
These are the ign_files were asking about in the comments section. Should we be ignoring everything for speed or something? or just leaving them so if someone were to 'enable' a driver later they wouldnt have to do anything here.